Core/include/Acts/EventData/detail/TransformationFreeToBound.hpp

```diff
@@ -27,9 +28,9 @@ namespace detail {
 ///
 /// @warning The position is assumed to be on the surface. If this is not
 ///          the case, the behaviour is undefined.
```

Comment:
Shouldn't that @warning be removed now, the behaviour is defined now - indicates that this PR is indeed the right direction.

Reply:
Gone now.